addon/mixins/routes/application.js

```diff
@@ -31,6 +31,13 @@ export default Ember.Mixin.create({
         outlet: outlet,
         parentView: parentView
       });
+
+      // Check if another modal exists then give it focus
+      Ember.run.later(function() {
+        var remainingViewElement = Ember.$('[role="dialog"]:last-child').closest('.ember-view');
```

Comment:
Without the `closest` call the modal will get the focus and we have to add some custom css to hide the border given to focused elements. Curiously, when the view itself has the focus, the border doesn't appear.